modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
```

Comment:
Are you going to include this module in an assembly? I'd love to be able to take a look at in in context and review the generated text.

Reply:
This PR is part of the `ztp-deploying-disconnected` assembly.

Reply:
Then you need to add the preview link to the first comment. (You have to go back and edit the comment after Netlify renders it, and it's best to link as close to the relevant change that you're making as possible.)

Reply:
Acked.